packages/ember-svg-jar/lib/inline-packer.js

```diff
@@ -65,6 +62,8 @@ class InlinePacker extends CachingWriter {
     saveToFile(
       path.join(outputPath, `index.js`),
       `import { importSync } from '@embroider/macros';\n` +
+        `export ${makeSafeFunctionName.toString()}` +
```

Comment:
Another idea that came to mind:
Instead if named exports, you could export an object.
const obj = {
  [name]: func,
  ...
}
export default obj;

Reply:
Well that's another good idea!